.github/workflows/eval.yml

```
@@ -0,0 +1,53 @@
+name: Eval
+
+on: pull_request_target
```

Nobody important here, but what about `push` as well?

---

There is some code in this commit that wouldn't trigger on push.

But I think we will a have a different action later to compute the baseline store paths:

Here some code:

```
on:
  push:
    branches:
      - main
      - staging-*
      - release-*
      - python-updates
```

---

I was thinking something like:
```suggestion
on: [push, pull_request_target]
```
So it'll run on any branch, not just the branches listed.

---

I would rather not do that because than you are duplicating ci if someone opens a pull request from a branch in the repository. Also the github action you see here will not do anything if it runs on a branch because of the checks it does.